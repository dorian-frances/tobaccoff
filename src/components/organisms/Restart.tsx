import { View } from 'react-native';
import CircleButton from '../atoms/buttons/CircleButton';
import SectionText from '../atoms/texts/SectionText';

type RestartProps = {};

const Restart = ({}: RestartProps) => {
  return (
    <View style={{ alignItems: 'center' }}>
      <View>
        <CircleButton />
      </View>
      <View style={{ marginTop: 10 }}>
        <SectionText text={'RedÃ©marrer le compteur'} fontSize={12} />
      </View>
    </View>
  );
};

export default Restart;
