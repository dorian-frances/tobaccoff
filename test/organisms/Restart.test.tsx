import { render } from '@testing-library/react-native';
import Restart from '../../src/components/organisms/Restart';
import React from 'react';

describe('<Restart/>', () => {
  it('should render correctly', () => {
    const component = render(<Restart />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
