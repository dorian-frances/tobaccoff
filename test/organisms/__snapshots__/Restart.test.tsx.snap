// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Restart/> should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
    }
  }
>
  <View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": true,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        [
          false,
          false,
          undefined,
        ]
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#FFFFFF",
            "borderRadius": 100,
            "elevation": 2,
            "height": 70,
            "justifyContent": "center",
            "width": 70,
          }
        }
        testID="circleButton"
      >
        <Text />
      </View>
    </View>
  </View>
  <View
    style={
      {
        "marginTop": 10,
      }
    }
  >
    <Text
      style={
        {
          "color": "#79747E",
          "fontFamily": "Montserrat-Medium",
          "fontSize": 12,
        }
      }
    >
      Red√©marrer le compteur
    </Text>
  </View>
</View>
`;
